tests/online/clone.c

```diff
@@ -276,6 +276,9 @@ static int cred_failure_cb(
 
 void test_online_clone__cred_callback_failure_return_code_is_tunnelled(void)
 {
+	git__free(_remote_url); _remote_url = NULL;
+	git__free(_remote_user); _remote_user = NULL;
```

Comment:
No need to `NULL` those variables, as you unconditionally set them to the result of `git__strdup` in the next statements. Same applies to the other test

Reply:
Right you are.  Fixed.